scripts/run-monitor-ci-tests.bash

```diff
@@ -76,7 +76,9 @@ fi
 # start the monitor-ci pipeline
 DEPLOY_PROD=false
 if [[ "${UI_BRANCH}" == "master" ]]; then
-	DEPLOY_PROD=false # TODO: change this to true when we're ready to depend on this script
+	# UI_BRANCH and DEPLOY_PROD are both neccesary to ensure that UI_BRANCH parameter's default in the monitor-ci pipeline doesn't deploy us to production unless started by this script.
+	# note: it is only possible to deploy the master branch of the UI repo to production
+	DEPLOY_PROD=true
```

Comment:
this ok @Asalem1 ?

Reply:
Whoops, should've clarified that I meant in terms of the comments on the PR.

> >Closes influxdata/idpe#10480 and influxdata/monitor-ci#215
> 
> The experiment from #1112 is finished, time to use the monitor-ci pipeline full time.
> 
> This PR removes all the "private" jobs from the UI CI.
> 
> TODO after merging this:
> 
> update github actions so cloud-e2e, smoke, oss-e2e aren't required (because now they don't exist). Instead, make monitor-ci-tests required for merging.
> Merge this first: influxdata/monitor-ci#224 ✅

Reply:
OH lol 😆